resources/views/filament/app/pages/sampling.blade.php

```diff
@@ -1,3 +1,10 @@
+<?php
+use App\Filament\App\Clusters\LocationLevels\Resources\FarmResource;
+
+$farmUrl = url(FarmResource::getUrl());
```

Comment:
no need to wrap in a url(), as the getUrl() will get a fully-qualified url for the page.